pkg/hsskey/hsskey.go

```diff
@@ -116,12 +116,12 @@ func getHssFromIpmi(verbose bool, verboseDangerous bool) ([][]uint8, error) {
 			continue
 		}
 
-		if verbose {
+		if warnings != nil {
 			msg := fmt.Sprintf("HSS Entry: Id=%s", id)
 			if verboseDangerous {
 				msg = msg + fmt.Sprintf(",Seed=%x", hss)
 			}
-			log.Print(msg)
+			io.WriteString(warnings, msg+"\n")
```

Comment:
Since you've changed the calling code to pass in `warnings` regardless of verbosity flags, this might be too spammy in boot logs.  I would suggest gating the whole log here on `verboseDangerous`, like you've done with similar logging in `GetHssFromFile`.

Reply:
Fair, good catch. Updated, ptal